bootstrap/namespaces/helmfile/values/namespaces-wc.yaml.gotmpl

```diff
@@ -1,21 +1,62 @@
 namespaces:
 - name: cert-manager
+  labels:
+    pod-security.kubernetes.io/audit: baseline
+    pod-security.kubernetes.io/audit-version: v1.25
+    pod-security.kubernetes.io/enforce: baseline
+    pod-security.kubernetes.io/enforce-version: v1.25
+    pod-security.kubernetes.io/warn: baseline
+    pod-security.kubernetes.io/warn-version: v1.25
 - name: fluentd
+  labels:
+    pod-security.kubernetes.io/audit: privileged
+    pod-security.kubernetes.io/audit-version: v1.25
+    pod-security.kubernetes.io/enforce: privileged
+    pod-security.kubernetes.io/enforce-version: v1.25
+    pod-security.kubernetes.io/warn: privileged
+    pod-security.kubernetes.io/warn-version: v1.25
 - name: kube-node-lease
 - name: kube-public
 - name: kube-system
   labels:
     admission.gatekeeper.sh/ignore: "true"
 - name: monitoring
+  labels:
+    pod-security.kubernetes.io/audit: privileged
+    pod-security.kubernetes.io/audit-version: v1.25
+    pod-security.kubernetes.io/enforce: privileged
+    pod-security.kubernetes.io/enforce-version: v1.25
+    pod-security.kubernetes.io/warn: privileged
+    pod-security.kubernetes.io/warn-version: v1.25
 - name: ingress-nginx
+  labels:
+    pod-security.kubernetes.io/audit: baseline
```

Comment:
Same comment here, as above. Would the Ingress Controller work in hostNetwork mode only with `baseline`?